pkg/controller/controller_test.go

```diff
@@ -1593,9 +1647,7 @@ func assertServiceInstanceRequestFailingError(t *testing.T, obj runtime.Object,
 	}
 	assertServiceInstanceReadyCondition(t, obj, readyStatus, readyReason)
 	assertServiceInstanceCondition(t, obj, v1alpha1.ServiceInstanceConditionFailed, v1alpha1.ConditionTrue, failureReason)
-	assertServiceInstanceCurrentOperationClear(t, obj)
```

Comment:
I would much rather see a new assert method created to assert that orphan mitigation has been started rather than pulling the asserts on current operation and reconciliation complete out of this assert method.

Reply:
I can do that; the scope of this function will change to represent non-orphan-mitigating request failure cases.